src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -216,32 +217,37 @@ function latent_heat_generic(T, LH_0, cp_diff)
 
 end
 
+abstract type Phase end
+struct Liquid <: Phase end
+struct Ice <: Phase end
+struct Generic <: Phase end
+
 """
-    sat_vapor_press_liquid(T)
+    sat_vapor_press(T, Liquid())
```

Comment:
- a more verbose name might be better (`saturation_vapor_pressure` is easier to understand).
- using `temp` instead of `T` (Julia convention has it that `T` is usually a type parameter)
- these methods can all share the one docstring

Reply:
- Agree with `saturation_vapor_pressure`
- So, temperature shows up a lot and I think that, while `T` is used in Julia for types, it may be more appropriate here to use `T` (or `temperature`) for temperature. My only hesitation on `temp` is confusion with temporary variables (which I usually like using `temp` for). I'd like to hear @tapios weigh in on this.
- Agree with shared docstring

Reply:
* Updating to ``saturation_vapor_pressure`` is good
* Prefer to avoid ``temp`` for ambiguity @charleskawczynski mentioned
* One may argue a few centuries of history of ``T`` for temperature may outweigh Julia conventions, unless there is real potential for confusion
* If we want to replace ``T``, use, e.g., ``Ta`` for atmospheric temperature, which is close to CMIP convention for this variable ``ta`` (but I prefer uppercase T because it is closer to how we write the equations)

Reply:
* If you change ``sat_`` to ``saturation_`` in the vapor pressure functions, I suggest you do the same for the humidity (``shum``) functions